content/blog/ways-to-interact-with-kubernetes-clusters-managed-by-hpe-ezmeral-container-platform.md

```diff
@@ -0,0 +1,180 @@
+
+| Screenshot      | Description |
+| ----------- | ----------- |
+| ![](https://github.com/helloezmeral/cdn/raw/main/K8s-Tenant-03.png)      | You can download the required binary here as well.  |
+
+### Installation of ```kubectl``` and ```kubectl-hpecp```
+#### Step 1: Make sure you have ```kubectl``` install.
+```bash
+# Download the latest version of kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+# And place it anywhere in your PATH:
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+```
+> Reference:
+> - [How to install and set up kubectl on Linux](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+> 
+
+#### Step 2: Install hpe kubectl plugin
+```bash
+# This link might subject to be changed, renew this link on HPE Ezmeral
+# Download kubectl-hpecp binary and untar the file
+wget https://bluedata-releases.s3.amazonaws.com/kubectl-epic/3.4/14/linux/kubectl-hpecp.star
+tar xf kubectl-hpecp.star
+# And place it anywhere in your PATH:
+sudo mv ./kubectl-hpecp /usr/local/bin
+```
+Check kubectl-hpecp is installed correctly.
+
+| Command | Screenshot      | 
+| ----------- | ----------- |
+| ``` kubectl plugin list ``` | ![](https://github.com/helloezmeral/cdn/raw/main/kubectl-plugin-list.png)        |
+| ``` kubectl hpecp -h ``` | ![](https://github.com/helloezmeral/cdn/raw/main/kubectl-hpecp-h.png)       |
+
+> Reference:
+> - [Using the HPE Kubectl Plugin](https://docs.containerplatform.hpe.com/53/reference/kubernetes/using-kubernetes/Using_the_HPE_Kubectl_Plugin.html)
+> - [Extend kubectl with plugins](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/)
+
+### Getting the ```kubeconfig``` file
+#### Using ```kubectl hpecp refresh``` command
+
+The ```kubectl hpecp refresh``` command gets the user a new Kubeconfig. Using the Kubeconfig, you can interact with Kubernetes through the HPE Ezmeral Container Platform.
+
+```bash
+kubectl hpecp refresh <ip_address, host alias, or hostname> --insecure --hpecp-user=<new_username> --hpecp-pass=<new_password>
+# Example
+kubectl hpecp refresh 172.16.10.41 --insecure --hpecp-user=your-username --hpecp-pass=your-pass
+kubectl hpecp refresh ez53-gateway.hpeilab.com --insecure --hpecp-user=your-username --hpecp-pass=your-pass
+kubectl hpecp refresh ez53-gateway.hpeilab.com --insecure
+```
+
+Download your Kubeconfig file, and define the path to the Kubeconfig file as a shell environment variable.
+
+![image](https://user-images.githubusercontent.com/72959956/117413580-bab71980-af48-11eb-808e-1f46f074451c.png)
+
+```bash
+# Example
+export KUBECONFIG="/home/hpeadmin/.kube/.hpecp/ez53-gateway.hpeilab.com/config"
+```
+
+#### Download the Kubeconfig file manually
+![image](https://user-images.githubusercontent.com/72959956/119962105-4b799600-bfd9-11eb-985d-2c867162902e.png)
+
+- Download your ```kubeconfig``` file, and define the Kubeconfig file as a shell environment variable

```

Comment:
Suggestion to rewrite the sentence as follows:

Download your ```Kubeconfig``` file, and define the path to the Kubeconfig file as a shell environment variable. 

For consistency, I suggest a capital "K" for the word "Kubeconfig file".
Please add a period at then end of the sentence. 
Also, this sentence should not be a bullet item.